keep-ui/app/topology/topology.tsx

```diff
@@ -0,0 +1,238 @@
+"use client";
+import React, { useCallback, useEffect, useState } from "react";
+import {
+  Background,
+  BackgroundVariant,
+  Controls,
+  Edge,
+  // MiniMap,
```

Comment:
forgotten?

Reply:
Left it as something to be added, I can remove if it makes more sense